components/checkbox.js

```diff
@@ -1,15 +1,34 @@
 const template = `
   <div class="c-chk">
-    <input class="c-chk__input" :id="name" type="checkbox" :checked="value" @change="$emit('input', $event.target.checked)" />
-    <label class="c-chk__label" :class="{'is-checked': value}" :for="name">
+    <input class="c-chk__input" :id="name" type="checkbox" :disabled="disabled===true" :checked="value" @change="change" />
```

Comment:
`disabled===true` let's stick to using truthy/falsey values in this case, you never know how the devs might be passing in these props

Reply:
Thing is default html attributes afaik cant receive truthy/falsey value instead then need to have whole comparion, so we shoiuld probably make this as a computed if its even going to work. This answers other observations aswell